book/src/transforms/sort.md

```diff
@@ -39,3 +39,31 @@ We can also use expressions:
 from employees
 sort [s"substr({first_name}, 2, 5)"]
 ```
+
+## Notes
+
+### Ordering guarantees
+
+Most DBs will persist through ordering most transforms; for example, you can
```

Comment:
I think it should be "ordering through" instead of "through ordering"

Reply:
lol thanks